service/conf/routes

```diff
@@ -208,3 +208,6 @@ POST   /v1/location/create                      @controllers.location.LocationCo
 PATCH  /v1/location/update                      @controllers.location.LocationController.updateLocation()
 POST   /v1/location/search                      @controllers.location.LocationController.searchLocation()
 DELETE /v1/location/delete/:locationId                      @controllers.location.LocationController.deleteLocation(locationId:String)
+
+#Initalization API
+POST   /v1/system/init                          @controllers.initialization.InitializationController.initalizeSystem
```

Comment:
Rename path to /v1/system/init/root/org. As in future, we may possibly extend to have other initialisations for other entities in the platform.